examples/substrate/vitalam-app/charts/vitalam-frontend/templates/deployment.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: app/v1
+kind: Deployment
+metadata: 
+  name: frontEnd
+  namespace: frontend-ns
+    labels: 
+      apps: vitalam-fe
+spec:
+  replicas: 1
+  selector: 
+    matchLabels: 
+      app: vitalam-fe
+  template:
+    metadata:
+      annotations:
+        checksum/service: {{ include (print $.Template.BasePath "/service.yaml") . | sha256sum }}
+      labels:
+        app: vitalam-fe
+    spec:
+      imagePullSecrets: 
+        - name: regcred
```

Comment:
no hardcoding in templates, you have to use the values.yaml file to define default values